### PR TITLE
Fixes #124: Store the linked project information relative to the project root.

### LINKED ISSUE #124: Store the linked project information relative to the project root
By doing this one could add this file to the repository and allow it to work with this on other environments (CI,lando,other devs) etc.. Otherwise each is force to link it again.

### PR DESCRIPTION
**Pull request by grasmash**
I've moved the ACLI filestore to be in a `.acquia-cli.yml` file in the project root. The `send_telemetry` setting has been moved to the Cloud filestore.

### CODE REVIEW

File: src/DataStore/YamlStore.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use Webmozart\KeyValueStore\ArrayStore;
+
+/**
+ * A key-value store backed by a PHP array.
+ *
+ * The contents of the store are lost when the store is released from memory.
+ *
+ * @since  1.0
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
```
Comment (danepowell) [inline src/DataStore/YamlStore.php:16]: webmozart?

Comment (grasmash) [reply src/DataStore/YamlStore.php:16]: Nope!

**Comment by danepowell** [comment]
No migration path for existing linked projects? I'm fine with that, but we should communicate it clearly.

**Comment by grasmash** [comment]
I'v added a migrate path for the send_telemetry preference.

**Pull request merged**